contracts/protocol/libraries/position/Position.sol

```diff
@@ -17,8 +17,8 @@ library Position {
         uint256 margin;
```

Comment:
Should Pack `quantity` `margin` `openNotional ` `lastUpdatedCumulativePremiumFraction ` to a single slot as well